src/AutoRest.CSharp/Mgmt/AutoRest/MgmtOutputLibrary.cs

```diff
@@ -137,6 +146,186 @@ private void RemoveOperations(CodeModel codeModel)
             }
         }
 
+        private void OmitOperationGroups(CodeModel codeModel, BuildContext<MgmtOutputLibrary> context)
```

Comment:
move all of this logic to a separate class to isolate it and make it easier to refactor later